public/pages/Rules/components/RuleEditor/DetectionVisualEditor.tsx

```diff
@@ -420,6 +437,10 @@ export class DetectionVisualEditor extends React.Component<
     ];
   };
 
+  private getTextareaHeight = (rowNo: number = 0) => {
+    return `${rowNo * 25 + 40}px`;
```

Comment:
Let's label the numbers `20` and `40`? and can we add a comment explaining why we chose those numbers?